index.html

```diff
@@ -3,22 +3,149 @@
+          <p class="secondary-text time-zone-reference">4:30am (NZST)</p>
+        </div>
+      </div>
+
+      <div class="teammate-avatar">
+        <img src="/images/junnie-small.png" alt="Junnie's avatar, it's a raccon">
+
+        <div class="teammate-details hidden">
+          <img
+            src="/images/junnie-large.png"
+            alt="Junnie's avatar in larger size."
+            class="expanded-avatar"
+          >
+          <p class="name">Junnie</p>
+          <p class="secondary-text location">Portland, United States</p>
+          <p class="secondary-text time-zone-reference">9:30am (PDT)</p>
+        </div>
+      </div>
+
+      <div class="teammate-avatar">
+        <img src="/images/febby-small.png" alt="Febby's avatar, it's a fox.">
+
+        <div class="teammate-info hidden">
+          <img
+            src="/images/febby-large.png"
+            alt="Febby's avatar in larger size."
+            class="expanded-avatar"
+          >
+          <p class="name">Febby</p>
+          <p class="secondary-text location">London, England</p>
+          <p class="secondary-text time-zone-reference">5:30pm (BST)</p>
+        </div>
+      </div>
+
+      <div class="teammate-avatar">
+        <img src="/images/augustin-small.png" alt="Augustin's avatar, it's a tiger.">
+
+        <div class="teammate-info hidden">
+          <img
+            src="/images/augustin-large.png"
+            alt="[teammate] avatar expanded"
+            class="expanded-avatar"
+          >
+          <p class="name">Augustin</p>
+          <p class="secondary-text location">Kathmandu, Nepal</p>
+          <p class="secondary-text time-zone-reference">10:15pm (NPT)</p>
+        </div>
+      </div>
+
+      <div class="teammate-avatar">
+        <img
+        src="/images/deercember-small.png" alt="Deercember's avatar, it's a deer.">
+
+        <div class="teammate-info hidden">
+          <img
+            src="/images/deercember-large.png" 
+            alt="Deercember's avatar in larger size."
+            class="expanded-avatar"
+          >
+          <p class="name">Deercember</p>
+          <p class="secondary-text location">Toronto, Canada</p>
+          <p class="secondary-text time-zone-reference">10:30pm (EDT)</p>
+        </div>
+      </div>
+
+      <div class="teammate-avatar hiring">
+        <p id="you">1+</p>
+        <div class="teammate-info hidden">
+          <p class="expanded-avatar">&#128075;</p>
+          <p class="name">You!</p>
+          <p class="secondary-text location">Anywhere</p>
+          <p class="secondary-text">We're 100% Remote</p>
+        </div>
+      </div>
+    </div>
 
-  <!-- Por aquí empieza el código. Con ese espacio alrededor el código es más legible y el doc más fácil de navegar. -->
+    <p class="navbar-cta navbar-item right" id="navbar-cta-text">We're hiring</p>
+  </nav>
 
+  <div id="hiring">
+    <div id="callAction">
+      <h3> We're Hiring </h3>
```

Comment:
When there are many nested divs or containers (which btw contain a lot of elements) within a large parent container, leave an empty line between containers the nested containers. 

This makes it more readable and easy to find where the parent container begins and ends. It also makes it easy to locate the beginning and end of each children container within the parent container.

E.g. Instead of this:
<div>
  <div>
    (Insert 3+ elements)
  </div>
  <div>
    (Insert 3+ elements)
  </div>
</div>

You do this:

<div>
  <div>
    (Insert 3+ elements)
  </div>

  <div>
    (Insert 3+ elements)
  </div>
</div>

Try it out in your own projects. You'll notice the difference.

Reply:
No se nota la diferencia del ejemplo,